scripts/buildtools/commands/verify-package.js

```diff
@@ -1,17 +1,32 @@
 exports.command = 'verify-package';
 exports.describe = 'Verifies that the NPM package has the correct format';
-exports.handler = function() {
+
+function verifyAuthJSVersion() {
```

Comment:
Future refactor idea: we can turn this script into a plugin of whatever bundler we will eventually stay with. Currently, we have build tools in this repo and how they are connected together is not very straight forward, it would be great to have one bundler as the source of truth, then other process can be built around it as loaders or plugins.

Seems like there is a way to link webpack and grunt together: https://webpack.js.org/guides/integrations/#grunt

Reply:
The `scripts/buildtools/commands` pattern is used in other repos in Okta, including okta-ui. I think the pattern is mean to solve the problem you describe, to have one place that links together the build tools.

We can switch to rollup, but I'm not sure how that would change the verifications we are doing here, which are on the final package, not any individual bundle.

Reply:
There is a valid argument against loaders/plugins which is that it makes it harder to migrate to a different build tool. If/when we switch to rollup we will need to find/build replacements for every webpack plugin. I think we should be thoughtful with our use of plugins, unless it is directly related to the specific build process and needs to be "in stream", we may be better off with a series of simple scripts which can be run independently.

Reply:
Yeah, as long as we have a pattern to follow to manage all script together, then we are in good place. My main concern is currently we have different build scripts, grunt, webpack, verify-package, locate in different places, it increases the difficult of understanding it and future upgrade.